resources_test.go

```diff
@@ -99,22 +118,18 @@ func TestBrowsersFolders(t *testing.T) {
 	}()
 
 	printBrowsers()
+	checkTableCorrect(`\| +BROWSER ID +\| +BROWSER DESCRIPTION +\|`, t)
+	checkTableCorrect(`\| +firefox +\| +Mozilla Firefox +\|`, t)
+}
 
-	pattern := `\| +BROWSER ID +\| +BROWSER DESCRIPTION +\|`
-	matched, err := regexp.Match(pattern, out.(*bytes.Buffer).Bytes())
-	if err != nil {
-		t.Error("Error with pattern match:", err)
-	}
-
-	pattern = `\| +firefox +\| +Mozilla Firefox +\|`
-	matched, err = regexp.Match(pattern, out.(*bytes.Buffer).Bytes())
-	if err != nil {
-		t.Error("Error with pattern match:", err)
-	}
-
-	if !matched {
-		t.Logf("Table didn't match properly:")
-		t.Logf("%v\n", out)
-		t.Errorf("should have matched %v", pattern)
-	}
+func TestPrintBrowsersApiError(t *testing.T) {
+	sitesResp := `{"error": "This is a bad thing"}`
+	ts := newTestServer("/clients.json", sitesResp, 600, t)
+	defer ts.Close()
+	baseURL = ts.URL
+	out = &bytes.Buffer{}
+	defer func() {
+		out = os.Stdout
+	}()
+	runErrorTest("Browsrs", t)
```

Comment:
This shouldn't be passing with this typo. Make sure this fails once you fix the `runErrorTest` method to use the string you pass in. (Currently it's not used)